ui/build/script.build.javascript.js

```diff
@@ -51,6 +51,7 @@ const builds = [
       },
       output: {
         file: resolve(`dist/quasar.common.js`),
+        exports: 'auto',
```

Comment:
I'd rather we not go through the upgrade hell of dev deps and testing all of the flavours of Quasar. This adds no benefit to the end user but adds possible failure points for us.
So please downgrade back all deps.

Reply:
Ok